Clarify persistent storage settings. (#385)
deviantony/docker-elk: README.md

The data stored in Elasticsearch will be persisted after container reboot but not after container removal.

In order to persist Elasticsearch data even after removing the Elasticsearch container, you'll have to mount a volume on

> your Docker host. Edit ```docker-compose.yml``` and add the path to persistent storage underneath the existing entry in the `elasticsearch` service declaration:

Reviewer: @seanfcarroll thanks for the suggestion! Your addition is correct but maybe a bit verbose. How about something more succinct like:

```suggestion
your Docker host. Add the following bind mount to the existing volumes of the `elasticsearch` service declaration:
```

Reviewer: Ping @seanfcarroll

```yml
elasticsearch:

  volumes:

>     - ./elasticsearch/config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml:ro

Reviewer: ```suggestion
    # replace <path to storage> with the path of the host directory to be mounted
```

>     - /path/to/storage:/usr/share/elasticsearch/data

Reviewer: ```suggestion
    - <path to storage>:/usr/share/elasticsearch/data
```

Reviewer 2: Perhaps to make it clearer for the uninitiated, you can use the phrase `local storage` instead of just `storage`. Thanks.
